### PR TITLE
Revert "Create admin rubric which allows to manually deposit USD" (#483)

### PR DESCRIPTION
**Pull request by Liapin**

### CODE REVIEW

File: app/models/currency.rb [hunk 1]
```diff
@@ -4,35 +4,28 @@ class Currency < ActiveYamlBase
   field :visible, default: true
```
Comment (yivo) [inline app/models/currency.rb]: @Liapin Pls revert the changes to be up to date with devel.

Comment (yivo) [reply app/models/currency.rb]: For this file.

**Review by yivo — CHANGES_REQUESTED** [review]

**Comment by Liapin** [comment]
@yivo done

**Review by yivo — APPROVED** [review]

**Comment by mod** [comment]
@yivo why reverting instead of creating a new withdraw?
I still think we need the admin section.

**Comment by yivo** [comment]
@mod We need to revert it since it is not needed anymore. It doesn't relate to withdraw. It relates to fiat deposit.

**Comment by mod** [comment]
@yivo we still need the ability just we need to create a Deposits.new instead of updating the balance, but as admin, I should be able to create a deposit for a customer.

**Comment by yivo** [comment]
@mod That's why we have #506 & #507 for it. The feature this PR contains is not needed anymore.

**Pull request merged**